app/main.py

```diff
@@ -1 +1,40 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            feeded = self.appetite
+            self.appetite = 0
```

Comment:
<img width="797" alt="image" src="https://github.com/mate-academy/py-feed-animals/assets/102595649/d392002e-05c4-4af7-8824-55f134699fc0">
So you do not need this code on line 15 and 16 . Or why did you add it ?)

Reply:
didn't fully understand the logic in the first time)